openlibrary/core/vendors.py

```diff
@@ -164,6 +164,25 @@ def _get_amazon_metadata(id_=None, id_type='isbn'):
     return _serialize_amazon_product(product)
 
 
+def split_amazon_title(full_title):
+    """Splits an Amazon title into (title, subtitle),
+    strips parenthetical tags.
+    :param str full_title:
+    :rtype: tuple
+    :return: (title, subtitle | None)
+    """
+    # strip parens
+    full_title = re.sub('\([^\)]*\)', '', full_title)
+    re_split_title = re.compile(
```

Comment:
It looks like this regex is _also_ removing parents; maybe let's flip this around? That would entirely remove this complex regex. So first split into title/subtitle, and then remove the parens on each. (`title_parts = full_title.split(': ', 1)`, then `trim`/parens regex)

Reply:
removing the complex regex :+1: I copied it from existing OL code, and while it looks like it is doing something with parenthesis, it doesn't seem to affect any examples I have found, so simplifying and making what it needs to do clearer is a good suggestion!